yarn.lock

```diff
@@ -1603,6 +1603,11 @@ bls-eth-wasm@^0.4.4:
   resolved "https://registry.yarnpkg.com/bls-eth-wasm/-/bls-eth-wasm-0.4.4.tgz#3d4c99f8ddee6df23e188dc756125268a0f4d525"
```

Comment:
shouldn't this block ^ be deleted? (no longer need bls-eth-wasm^0.4.4)

Reply:
That's a good point, is yarn missbehaving?